tests/installation/partitioning_warnings.pm

```diff
@@ -50,7 +49,7 @@ sub run {
         assert_screen 'partition-warning-no-efi-boot';
         wait_screen_change { send_key 'alt-y' };    # yes
     }
-    if (is_storage_ng && !check_screen('partitioning-edit-proposal-button', 0)) {
+    if (is_storage_ng && check_screen('partitioning-edit-proposal-button', 0)) {
```

Comment:
We should get rid of this, as bug is resolved and there will be no warning for missing swap, as we have new checkbox for that. So this change doesn't make sense anymore...